app/models/nearby_facility.rb

```diff
@@ -67,6 +85,15 @@ def make_linestring(polygon)
       polygon.map { |point| "#{point[1]} #{point[0]}" }.join(',')
     end
 
+    def parse_location(response_json)
+      response_json.try(:[], 'resourceSets')
```

Comment:
Wouldn't `dig('resourceSets', 'resources'....)` work here as well? I think that might be a lot more readable

Reply:
Does that work when there are arrays involved?

Reply:
@whuang85 maybe `dig(:resourceSets).first.dig(:resources).first.dig(:point, :coordinates)`? Same idea with the `first`s you have here.